IPython/core/application.py

```diff
@@ -272,18 +280,30 @@ def _ipython_dir_changed(self, change):
                 self.log.error("couldn't create path %s: %s", path, e)
         self.log.debug("IPYTHONDIR set to: %s" % new)
 
-    def load_config_file(self, suppress_errors=True):
+    def load_config_file(self, suppress_errors=IPYTHON_SUPPRESS_ERRORS):
         """Load the config file.
 
         By default, errors in loading config are handled, and a warning
         printed on screen. For testing, the suppress_errors option is set
         to False, so errors will make tests fail.
+
+        `supress_errors` default value can be change to `False` by setting the
+        IPYTHON_SUPPRESS_ERRORS environment variable to empty string, or `True`
+        by setting the IPYTHON_SUPPRESS_ERRORS environment variable to anythong
+        else. Leaving IPYTHON_SUPPRESS_ERRORS unset will leave
```

Comment:
I know, I hadn't realised there were three possible states when I first commented. I think I'd go for '1' = True, '0' = False, and empty/undefined = default, then. Not sure what it should do if it's set to another value.